actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -26,7 +26,42 @@ module TagHelper
       PRE_CONTENT_STRINGS[:textarea]  = "\n"
       PRE_CONTENT_STRINGS["textarea"] = "\n"
 
+      class TagBuilder #:nodoc:
+        include TagHelper
```

Comment:
I see it a bit more complicated then that - `content_tag_string` is used by `FormOptionsHelperTest`. I see 3 options:
1. Delegate `content_tag_string` to `builder` i.e.
```
    module TagHelper
        delegate :content_tag_string, to: :tag_builder
        ...
    end
```
Sounds most reasonable, but feels a bit weird. 

2. Use it from builder:
```
      def options_for_select(container, selected = nil)
          ...
          tag.content_tag_string(:option, text, html_attributes)
      end
```
or
3. That would means quite a lot of changes and would potentially slow down rendering a bit, but feels most right:
```
      def options_for_select(container, selected = nil)
          ...
          tag.option text html_attributes
      end
```